scss/core/_variables.scss

```diff
@@ -560,6 +560,22 @@ $level-5-box-shadow:          0 1.25px 2.5rem rgb(0 0 0 / 15%), 0 .5rem 2.5rem r
   }
 }
 
+$levels: (
```

Comment:
Are these variables specific to shadows, or are they intended to be more generic? If they are shadow-specific, should their names reflect that, e.g., `$box-shadow-levels` and `$box-shadow-sides` or similar?

Reply:
I think it's worth leaving `levels` as a working tool (maybe in the future you will have to use levels for cycles or new conditions)